cli/index.ts

```diff
@@ -71,33 +71,34 @@ const OPTIONS: OptionConfigs = {
   auth: {
     id: "au",
     description:
-      "Include built-in auth features (deprecated option, always true in newest starter code)",
```

Comment:
What I'm understanding about the "deprecated option" stuff is that users had the ability to opt out of auth and file storage in the deprecated starter code, but in the newest starter code, we want auth and file storage to be mandatory features. If we wanna follow through with what's being suggested, I think that means we can go ahead and remove those two options entirely...

... but honestly, I prefer giving users the option to still make a choice there. 
My suggestion would be to make the default values `true` for those two prompts (and to point this out, like we do for directory and branch), and still leave users with the option to opt out if they wish to.

Thoughts?